docker-compose.sample.yml

```diff
@@ -86,6 +86,9 @@ services:
     volumes:
       - ${VOLUMES_ROOT_PATH}/mail:/var/mail
       - ${VOLUMES_ROOT_PATH}/traefik/acme:/etc/letsencrypt/acme
+      # Comment out the line below, when you want whitelist some IP Addresses or domains in Postfix (please check the documentation)
```

Comment:
I personally often frustrated by an advice to check documentation, when it's not immediately clear what documentation it is and where to find it.

Reply:
Ok. I updated the path

Reply:
I think you meant "Uncomment the line below" instead of "Comment out the line below" because its already commented out

Reply:
Fixing

Reply:
Fixed.

Reply:
May I suggest "please see 'Whitelist Hosts/IP Addresses In Postfix' in README.md for more info". It makes where to look explicit, and also make it clear that we are talking about this project's README.md.

Reply:
I updated the given part that you mentioned.